README.md

```diff
@@ -266,6 +267,11 @@ The Asana section ID in the Asana Project
 Comma-separated IDs of Asana tags to be added to the task i.e. https://app.asana.com/0/1208613272217946/
 ### `asana-collaborators`
 Comma-separated Asana user IDs to be added as collaborators to the task
+### `asana-assignee`
+GID of user to assign the task to
+### `asana-task-custom-fields`
+Asana task custom fields hash, encoded as a JSON string i.e. '{"XXXXX":"YYYYY"}'
+
 * Note: you can use https://app.asana.com/api/1.0/users/me to find your ID. Replace `me` with an email to find someone else's
```

Comment:
Minor: this note was meant to go along with "Comma-separated Asana user IDs." Now that we are referring to user IDs in multiple places, we should probably just point to the user map repo.

Reply:
Added a note under the "Get Asana user ID" section, as that relies on the user mapping.